package.json

```diff
@@ -66,7 +66,7 @@
     "devtron": "^1.4.0",
     "electron-log": "^2.2.6",
     "electron-store": "^1.1.0",
-    "emerald-js": "git+https://github.com/ETCDEVTeam/emerald-js.git#v0.1.3",
+    "emerald-js": "git+https://github.com/ETCDEVTeam/emerald-js.git#v0.1.4",
```

Comment:
this depends on a new version of emerald-js we still need to tag

Reply:
done!